08-stripping.md

```diff
@@ -1,55 +0,0 @@
----
```

Comment:
[We do](https://github.com/lhcb/second-analysis-steps/pull/19/commits/c3ebef0475d04d4191ad655818ed42587076823c), as it's a dupe of the [LHCb data flow lesson](https://lhcb.github.io/first-analysis-steps/07-dataflow.html) in first-ana (which also contains more information).